src/pages/T4.js

```diff
@@ -0,0 +1,60 @@
+const { css } = require('emotion')
+const { theme } = require('../styles.js')
+const { html } = require('../utils.js')
+const Layout = require('../components/Layout.js')
+const ErrorList = require('../components/ErrorList.js')
+const LogoutLink = require('../components/LogoutLink.js')
+const SummaryTable = require('../components/SummaryTable.js')
+const ButtonLink = require('../components/ButtonLink.js')
+const ValidationError = require('../components/forms/ValidationError.js')
+const Checkbox = require('../components/forms/Checkbox.js')
+const Button = require('../components/forms/Button.js')
+
+const t4CSS = css`
+  position: relative;
+
+  > div {
+    margin-bottom: ${theme.space.xl};
+  }
+`
+
+const changesButtonCSS = css`
+  width: 200px;
+  background-color: pink;
+`
+
+const consentButtonCSS = css`
+  width: 200px;
+  float: right;
+`
+
+//todo How to get t4 info? Where does it get injected from? user object passed with an array of t4s?
+//is this passed in to the Dashbord object which then passes it to summary table?
+const t4Data = ({ employerName, year, box12, box14, box22 }) => {
+  return [
+    { key: 'Employer Name', value: employerName },
+    { key: 'Year', value: year },
+    { key: 'Box12', value: box12 },
+    { key: 'Box14', value: box14 },
+    { key: 'Box22', value: box22 },
+  ]
+}
+
+const T4 = ({ data = {}, errors = {} }) =>
+  html`
+    <${Layout}>
+      <div class=${t4CSS}>
+        <${LogoutLink} />
+        <h1>Income</h1>
+        <div>
+        <img src='/t4.png' />
+        </div>
+
+<${SummaryTable} title="Income Data" rows=${t4Data(data.income)} ifEditable=${false} //>
+<${ButtonLink} href="/confirmation" style=${changesButtonCSS}>I need to make changes<//>
+<${ButtonLink} href="/confirmation" style=${consentButtonCSS}>This information is accurate<//>
```

Comment:
The GOV.UK button guidance (https://design-system.service.gov.uk/components/button/#how-it-works) usually has succinct button text, and it also says "Align the primary action button to the left edge of your form." 

So let's keep the "positive" action on the left hand side.